crates/matrix-sdk-base/src/store/integration_tests.rs

```diff
@@ -1611,18 +1503,6 @@ impl StateStoreIntegrationTests for DynStateStore {
 #[allow(unused_macros, unused_extern_crates)]
 #[macro_export]
 macro_rules! statestore_integration_tests {
-    (with_media_tests) => {
```

Comment:
Same here, since there's no more macro rule for `with_media_tests`, we can probably inline the `(@inner)` expanded block into the `()` rule.